src/components/TableOfContents/index.tsx

```diff
@@ -65,43 +57,29 @@ const TableOfContents = ({
   }
 
   return (
-    <Flex
-      direction="column"
-      align="start"
-      gap={4}
-      hideBelow={lgBp}
-      as="aside"
-      position="sticky"
-      top="19" // Account for navbar
-      p={4}
-      pe={0}
-      maxW="25%"
-      minW={48}
-      height={calc.subtract("100vh", "80px")}
-      overflowY="auto"
+    <aside
+      className={cn(
+        "sticky top-19 hidden h-[calc(100vh-80px)] min-w-48 max-w-[25%] flex-col items-start gap-4 overflow-y-auto p-4 pe-0 lg:flex",
```

Comment:
<img width="467" alt="image" src="https://github.com/user-attachments/assets/9d8178a6-7cc9-492b-b9db-3ee8ad70e707">

Per discussion in Discord, noted that the `max-w-[25%]` isn't actually aligned with the DS. Looks like we use a fixed with of 16rem, or `w-64`

```suggestion
        "sticky top-19 hidden h-[calc(100vh-80px)] w-64 flex-col items-start gap-4 overflow-y-auto p-4 pe-0 lg:flex",
```

Reply:
should this be part of this PR? there may be potential about side effects.

Reply:
Can easily break it off as a separate DS task, sure.. don't see it as blocking. 

The regression to the scroll-tracking noted above I would still rather we fix before merging though.

Reply:
Good call. I'll do that in a separate PR to discuss potential issues with the design and move this forward.